.github/workflows/validate_packs.yml

```diff
@@ -352,12 +352,10 @@ jobs:
         run: |
           URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/$PR_NUMBER/files"
           cd utils && yarn dashboard-helper $URL
-
       - name: Comment
-        if: steps.helper.outputs.comment != ''
+        if: env.comment != ''
         env:
           PR_NUMBER: ${{ steps.get_pr_number.outputs.pr-number }}
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          COMMENT: ${{ steps.helper.outputs.comment }}
         run: |
-          gh pr comment "$PR_NUMBER" --body "$COMMENT"
+          gh pr comment "$PR_NUMBER" --body "env.comment"
```

Comment:
To keep this secure and not allow any injection attacks, let's keep the env declaration. 
```
env:
  PR_NUMBER: ${{ steps.get_pr_number.outputs.pr-number }}
  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  COMMENT: ${{ env.comment }}
run: |
  gh pr comment "$PR_NUMBER" --body "$COMMENT"
```
This will help us avoid being vulnerable to things like [word splitting](https://github.com/koalaman/shellcheck/wiki/SC2086)